microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md

```diff
@@ -175,16 +176,18 @@ A wildcard prefix is used in origin strings of the `url_handlers` member to matc
 
 For example, the `url_handlers` member value is set to `*.contoso.com` matches `tenant.contoso.com` and `www.tenant.contoso.com`, but doesn't match `contoso.com`.  
 
-<!-- Hold for future release -->  
-<!--  ## Window Controls Overlay for installed desktop web apps  
+ 
+## Window Controls Overlay for installed desktop web apps  
 
 To create an immersive title bar similar to a native app for your desktop installed web app.  The **Window Controls Overlay** feature  completes the following actions.  
     
 1.  Removes the system reserved title bar.  It usually spans the width of the client frame.  
 1.  Replaces it with an overlay.  It contains just the critical system required window controls necessary for a user to control the window itself.  
     
 After it provides an overlay, the entire web client area is available for you to use.  This feature includes a manifest update.  It provides ways for you to determine the size and position of the overlay to help you arrange content.  
-    
+
+To preview Window Controls Overlay in Microsoft Edge on Windows, navigate to [Turn on experimental features](#turn-on-experimental-features) and turn on **Desktop PWA Window Controls Overlay**. 
```

Comment:
Ah good catch - the doc guidance is to have any instructions like this to appear right before the example code so this is the appropriate spot for it. I included Windows 10 rather than desktop OSs and can modify that as we roll out to other platforms in 92.